src/main/java/org/prebid/server/auction/VideoStoredRequestProcessor.java

```diff
@@ -387,13 +387,16 @@ private ExtRequest createExtRequest(BidRequestVideo videoRequest) {
             durationRangeSec = videoRequest.getPodconfig().getDurationRangeSec();
         }
 
-        PriceGranularity updatedPriceGranularity = PriceGranularity.createFromString("med");
+        PriceGranularity updatedPriceGranularity;
```

Comment:
1. This variable can be inlined.
2. Pls point me the difference with prior code? i don't see the difference :(